app/generators/checkbox_generator.rb

```diff
@@ -28,18 +28,20 @@ def answer_options(question)
     safe_join(body)
   end
 
-  def question_options(elem_id)
+  def question_options(question, elem_id)
+    previous_value = question[:previous_response]&.[](elem_id)
     {
       type: 'checkbox',
       id: elem_id,
       name: answer_name(elem_id),
-      value: true
+      value: true,
+      checked: previous_value.presence || nil
     }
   end
```

Comment:
What about the otherwise option for checkboxes? I don't think it's being prefilled if someone selects "otherwise" for a checkbox?

Reply:
yes, good point, missed it